GameServer/gameobjects/GamePlayer.cs

```diff
@@ -15953,6 +15953,11 @@ public GamePlayer Bodyguard
 					: null;
 			}
 		}
-		#endregion
+
+        /// <summary>
+        /// This property is use for the assassinate RA
+        /// </summary>
+        public bool StayStealth { get; set; }

```

Comment:
I'm not sure this is really necessary. Semantically it does not belong here.

Reply:
This is the only way I found to make the player using Assassinate skill, keep being invisible after using the skill. If I remove that part, the player will no longer be invisible to other player. Or is it supposed to be what should happen ?

Reply:
Regarding Shadowstrike, do you need me to do the patch right now for players to use Backstab 2 instead of Perforate Artery if they have a low specialization in Critical Strike ? Or will you rather accept tu merge this PR and I will do the patch later ?

Reply:
For Shadowstrike I would accept it as is. As I see it is now like Assassinate, but with with PA/BS at the end. On Live you need to "cast" the ability while being stealthed. There is no cast animation and the cast time seems to be fixed (10 seconds). 

StayStealth has to be removed. I know it seems minor, however you probably see how big GameLiving/GamePlayer already are and adding another property that belongs to only one ability does not help (in there are more examples for this already :disappointed: ). I'd say even a partly broken Assassinate ability would be better. Maybe you can use the existing `GamePlayer.Stealth(bool)` :cry:  or plain remove Assassinate. You probably need `GamePlayer.Stealth(bool)` for Shadowstrike as well.